sql/core/benchmarks/IntervalBenchmark-results.txt

```diff
@@ -1,25 +1,29 @@
-Java HotSpot(TM) 64-Bit Server VM 1.8.0_231-b11 on Mac OS X 10.15.1
-Intel(R) Core(TM) i7-4850HQ CPU @ 2.30GHz
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_65-b17 on Mac OS X 10.14.6
```

Comment:
So old jdk? ;-)

Reply:
updated and regen-ed with both new jdk8 and jdk11, thanks.